exotic/api/colab.py

```diff
@@ -370,10 +370,13 @@ def make_inits_file(planetary_params, image_dir, output_dir, first_image, targ_c
             "Observing Notes": "%s",
 
             "Plate Solution? (y/n)": "y",
-            "Align Images? (y/n)": "y",
+            "Add Comparison Stars from AAVSO? (y/n)": "y",
 
             "Target Star X & Y Pixel": %s,
-            "Comparison Star(s) X & Y Pixel": %s
+            "Comparison Star(s) X & Y Pixel": %s,
```

Comment:
This is a little clunky since some strings are surrounded with quotes and some are not. I see why and that it works (`null` being subbed into a multiline string) but it creates a disjoint appearance while reading code. 

Just a comment on maintainable code, nothing to strictly fix here.